lnd.go

```diff
@@ -333,6 +333,32 @@ func Main(cfg *Config, lisCfg ListenerCfg, interceptor signal.Interceptor) error
 		return err
 	}
 
+	// Create a new RPC interceptor that we'll add to the GRPC server. This
```

Comment:
Realized I need to do some deeper refactor to make this work. We need to block calls to the walletunlocker before becoming active. Do not review this commit yet.

Reply:
Should be fixed now.